common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java

```diff
@@ -100,7 +102,7 @@ public void onNpcDialogue(NpcDialogEvent e) {
             scheduledAutoProgressKeyPress = null;
         }
 
-        if (autoProgress) {
+        if (autoProgress && !e.isBlocking()) {
```

Comment:
Either I'm confused, or you are confused. Pressing "sneak" is only relevant for blocking NPC dialogues, right? The ones that are not blocking is just announced to the chat.

So the autoProgress scheme in NpcDialogueOverlayFeature is only releavant for `e.isBlocking()`. So I think this test is still backwards.

Reply:
Blocking is a weird word in this scenario. I think of blocking as "this dialouge blocks the user from progressing by shift". You can think of it as "this dialogue blocks progress until shift"

Reply:
perhaps this is not that good of a name after all?

Reply:
Ok, seems we interpret "blocking" completely different here then! Yes, to  me, it "obviously" meant that progress was blocked, waiting for shift. :)

`needsConfirmation`? Is that ok? So the "normal" NPC dialogues that require you to press shift has `needsConfirmation` true?